hadoop-ozone/common/src/main/proto/OzoneManagerProtocol.proto

```diff
@@ -446,6 +459,22 @@ message BucketArgs {
     repeated hadoop.hdds.KeyValue metadata = 7;
 }
 
+message OzoneObj {
+  enum ObjectType {
+    VOLUME = 1;
+    BUCKET = 2;
+    KEY = 3;
+  }
+
+  enum StoreType {
+    OZONE = 1;
+    S3 = 2;
+  }
+  required ObjectType resType = 1;
+  required StoreType storeType = 2;
```

Comment:
Should we have a default StoreType=OZONE?

Reply:
done